packages/ckeditor5-utils/src/splicearray.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/splicearray
+ */
+
+/**
+ * Function that splices Arrays and could be used instead of Array.prototype.splice as it results
+ * into "Maximum call stack size exceeded" when dealing with very big number.
```

Comment:
```suggestion
 * Splices one array into another. To be used instead of `Array.prototype.splice` as the latter may
 * throw "Maximum call stack size exceeded" when passed huge number of items to insert.
```